starter-code/src/sequencer.js

```diff
@@ -1,68 +1,128 @@
-var LetterSequence = function(){}
+class LetterSequence {
+  createSequence(sequence) {
+    const characters = sequence.split("");
```

Comment:
You can use the spread operator instead of `split`; like this: `const characters = [...sequence]`.

By the way, great use of `const` 👍 .